samples/drivers/ipm/ipm_esp32/sample.yaml

```diff
@@ -2,7 +2,10 @@ sample:
   name: ESP32 IPM Sample
 tests:
   sample.ipm.ipm_esp32:
-    platform_allow: esp32_devkitc_wroom esp32_devkitc_wrover
+    platform_allow:
+      - esp32_devkitc_wroom/esp32/procpu
+      - esp32_devkitc_wrover/esp32/procpu
     tags:
       - samples
       - ipm
+    skip: true
```

Comment:
@nordicjm Skipping this test for now. Will enable again once the sample is converted to sysbuild as required.